core/src/main/java/io/dekorate/AbstractKubernetesHandler.java

```diff
@@ -86,7 +87,10 @@ protected void addDecorators(String group, C config) {
     if (Strings.isNotNullOrEmpty(config.getServiceAccount())) {
       resources.decorate(new ApplyServiceAccountNamedDecorator(config.getName(), config.getServiceAccount()));
     }
-    resources.decorate(group, new ApplyImagePullPolicyDecorator(config.getImagePullPolicy()));
+
+    if (config.getImagePullPolicy() != ImagePullPolicy.IfNotPresent) {
```

Comment:
Can't we just drop it completely?

I am probably wrong because I am thinking about this from a Quarkus perspective where the `ApplyImagePullPolicyDecorator` is always applied.

Reply:
No, we can't or we'll break it on dekorate side.

Reply:
:+1: